include/klee/Support/QueryLog.h

```diff
@@ -10,9 +10,8 @@
 #ifndef KLEE_OPT_LOGGINGSOLVER_H
 #define KLEE_OPT_LOGGINGSOLVER_H
 
-#include "klee/Expr.h"
-
 #include <vector>
+#include "klee/Expr/Expr.h"
```

Comment:
Again. Include order is wrong ( http://llvm.org/docs/CodingStandards.html#include-style )